### PR TITLE
New EMG widget created

### PR DESCRIPTION
**Pull request by cfausn**
I think this is stable but I still have to do some more testing. For the most part it's working though!

### CODE REVIEW

File: OpenBCI_GUI/OpenBCI_GUI.pde [hunk 1]
```diff
@@ -38,7 +38,7 @@ import java.util.Random;
 import java.awt.Robot; //used for simulating mouse clicks
 import java.awt.AWTException;
 
-import gifAnimation.*;
+//import gifAnimation.*;
```
Comment (aj-ptw) [inline OpenBCI_GUI/OpenBCI_GUI.pde]: please uncomment

File: OpenBCI_GUI/OpenBCI_GUI.pde [hunk 2]
```diff
@@ -62,7 +62,7 @@ final int NCHAN_GANGLION = 4;
 
 boolean hasIntroAnimation = false;
 PImage cog;
-Gif loadingGIF;
+//Gif loadingGIF;
```
Comment (aj-ptw) [inline OpenBCI_GUI/OpenBCI_GUI.pde]: please uncomment

File: OpenBCI_GUI/OpenBCI_GUI.pde [hunk 3]
```diff
@@ -321,8 +321,8 @@ void setup() {
   logo_blue = loadImage("logo_blue.png");
   logo_white = loadImage("logo_white.png");
   cog = loadImage("cog_1024x1024.png");
-  loadingGIF = new Gif(this, "OBCI-6.gif");
-  loadingGIF.loop();
+  //loadingGIF = new Gif(this, "OBCI-6.gif");
```
Comment (aj-ptw) [inline OpenBCI_GUI/OpenBCI_GUI.pde]: please uncomment

File: OpenBCI_GUI/OpenBCI_GUI.pde [hunk 4]
```diff
@@ -877,7 +877,7 @@ void systemDraw() { //for drawing to the screen
     //make out blink the text "Initalizing GUI..."
     pushStyle();
     imageMode(CENTER);
-    image(loadingGIF, width/2, height/2, 128, 128);//render loading gif...
+    //image(loadingGIF, width/2, height/2, 128, 128);//render loading gif...
```
Comment (aj-ptw) [inline OpenBCI_GUI/OpenBCI_GUI.pde]: please uncomment

File: OpenBCI_GUI/W_EMG.pde [hunk 5]
```diff
@@ -0,0 +1,1332 @@
+
+////////////////////////////////////////////////////
+//
+//    W_template.pde (ie "Widget Template")
```
Comment (aj-ptw) [inline OpenBCI_GUI/W_EMG.pde]: could you add a header to describe the file? this is awesome!

Comment (cfausn) [inline OpenBCI_GUI/W_EMG.pde]: yeah oops I was planning on uncommenting before the push but forgot to. Also I found some bugs so I will actually need some more time to work on this stuff anyway. Gonna work today!

**Comment by cfausn** [comment]
This should be ready to merge!

**Comment by aj-ptw** [comment]
Ok I'm gonna check it out!

**Comment by cfausn** [comment]
Do not merge this yet!! I'm almost done with the OpenBionics widget and I need to save my progress, but it's not ready for a merge. Will be done hopefully by the end of the night

**Comment by cfausn** [comment]
Okay this push has OpenBionics and EMG functionality! Should be working, but the code is a bit messy and the UI could use a bit of improvement. Let me know what you guys think!!

**Pull request merged**